omf-python/src/element.rs

```diff
@@ -31,4 +32,51 @@ impl PyElement {
     fn geometry(&self) -> PyGeometry {
         PyGeometry(self.0.geometry.clone())
     }
+
+    #[getter]
+    fn color(&self) -> Option<PyColor> {
+        self.0.color.map(PyColor)
+    }
+}
+
+#[pyclass(name = "Color")]
+pub struct PyColor(pub Color);
+
+#[pymethods]
+impl PyColor {
+    const RED: usize = 0;
+    const GREEN: usize = 1;
+    const BLUE: usize = 2;
+    const ALPHA: usize = 3;
+
+    #[new]
+    pub fn new(red: u8, green: u8, blue: u8, alpha: u8) -> Self {
+        PyColor([red, green, blue, alpha])
+    }
+
+    fn __eq__(&self, other: &Self) -> bool {
+        self.0 == other.0
+    }
+
+    #[getter]
+    fn red(&self) -> u8 {
+        self.0[Self::RED]
+    }
+
+    #[getter]
+    fn green(&self) -> u8 {
+        self.0[Self::GREEN]
+    }
+    #[getter]
+    fn blue(&self) -> u8 {
+        self.0[Self::BLUE]
+    }
+    #[getter]
+    fn alpha(&self) -> u8 {
+        self.0[Self::ALPHA]
+    }
+
+    fn as_list(&self) -> [u8; 4] {
```

Comment:
I wonder if Colors should just be exposed to python as an array of 4 u8 (similar to what we did with vertices) so no PyColor class.

Reply:
I did this mainly to document the positions, i.e. reg, green, blue, alpha.

Reply:
That's the normal convention though. If you're going to pass it to something in python to do with color manipulation it'll need it to be an array anyway.